CHANGELOG.md

```diff
@@ -1,4 +1,44 @@
-# Changelog
+# CHANGELOG
+
+## 1.25.0 (2022-04-11)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- [Tracking] Introduce a new fluent API `mlflow.last_active_run()` that provides the most recent fluent active run (#5584, @MarkYHZhang)
+- [Tracking] Add `experiment_names` argument to the `mlflow.search_runs()` API to support searching runs by experiment names (#5564, @r3stl355)
+- [Tracking] Add a `description` parameter to `mlflow.start_run()` (#5534, @dogeplusplus)
+- [Tracking] Add `log_every_n_step` parameter to `mlflow.pytorch.autolog()` to control metric logging frequency (#5516, @adamreeve)
+- [Tracking] Log `pyspark.ml.param.Params` values as MLflow parameters during PySpark autologging (#5481, @serena-ruan)
+- [Tracking] Add support for  `pyspark.ml.Transformer`s to PySpark autologging (#5466, @serena-ruan)
+- [Tracking] Add input example and signature autologging for Keras models (#5461, @bali0019)
+- [Models] Add support for logging models to preexisting destination directories (#5572, @akshaya-a)
```

Comment:
Nit: Can we move this entry lower in the `[Models]` subsection since it's a relatively small improvement?

Reply:
moved